tests/bunit.tests/Rendering/TestRendererTest.cs

```diff
@@ -91,7 +91,7 @@ public async Task Test010()
 
 		cut.RenderCount.ShouldBe(1);
 
-		await cut.Instance.Trigger();
+		await cut.InvokeAsync(() => cut.Instance.Trigger());
 
 		cut.RenderCount.ShouldBe(2);
```

Comment:
```suggestion
		var assertion = cut.WaitForAssertionAsync(() => cut.RenderCount.ShouldBe(2)); 
		await cut.InvokeAsync(() => cut.Instance.Trigger());
		await assertion;
```

This should work, right?

Reply:
It's a bit ugly, obviously, basically reversing the order of assertion and invocation.

Reply:
Yeah it does work